lib/SILOptimizer/Transforms/SILMem2Reg.cpp

```diff
@@ -1807,8 +1817,12 @@ void StackAllocationPromoter::run() {
   if (asi->getType().isOrHasEnum()) {
     for (auto it : initializationPoints) {
       auto *si = it.second;
-      auto src = si->getOperand(0);
-      valuesToComplete.push_back(src);
+      auto stored = si->getOperand(CopyLikeInstruction::Src);
+      valuesToComplete.push_back(stored);
+      if (lexicalLifetimeEnsured(asi)) {
+        auto lexical = getLexicalValueForStore(si, asi);
```

Comment:
Will `lexical` always be non-"null" here?  If a guaranteed lexical value is `store_borrow`'d to this alloc stack, it seems like it would be null.  Or maybe it doesn't matter if `OSSALifetimeCompletion` can complete the lifetime of `SILValue()`.  (If it matters, maybe `if (auto lexical = getLexicalValueForStore(is, asi)) { valuesToComplete.push_back(lexical); }` would work.)

Reply:
Done. Thanks!